.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -43,8 +31,6 @@ fun gradleParameters(os: OS = OS.linux, daemon: Boolean = true, isContinue: Bool
         if (daemon) "--daemon" else "--no-daemon",
         if (isContinue) "--continue" else "",
         """-I "%teamcity.build.checkoutDir%/gradle/init-scripts/build-scan.init.gradle.kts"""",
-        java7Homes[os]!!,
-        java9Homes[os]!!,
```

Comment:
If we remove this, does this break the release branch?

Reply:
Yes we do. My plan is we do the migration this after 5.0 release activity because I really hate these `java7Home` `java9Home` stuff. WDYT?

Reply:
That is fine with me. Note that we need to wait for 4.10.3 as well... I don't know how that will be released though...